.github/actions/github-dependabot-check/action.yml

```diff
@@ -0,0 +1,50 @@
+name: PR author and label check
+description: Checks PR author and label, using PR that led to merge on push
+
+inputs:
+  gh-token:
+    description: 'GitHub token with content read permission'
+    required: true
+  actor:
+    description: 'Check if the pull request was created by this actor'
+    required: false
```

Comment:
maybe you want to add dependabot as a default if dependabot is the main/initial use case?

then if you rename the action to something more generic it's fine for me to merge